build-system/babel-plugins/babel-plugin-transform-amp-asserts/index.js

```diff
@@ -65,7 +67,12 @@ module.exports = function(babel) {
         // don't remove.
         const args = path.node.arguments[0];
         if (args) {
-          path.replaceWith(args);
+          if (parenthesized) {
+            path.replaceWith(t.parenthesizedExpression(args));
```

Comment:
It seems when i have a node that looks like `/** @type {x} */ (dev().assert(2));` and we remove the dev assert call, the parenthesis get's removed. which breaks the cast. I could merge the behavior of `babel-plugin-transform-parenthesize-expression/index.js` into here but wanted the behavior to be stand alone. LMK if i'm doing anything wrong.

Reply:
Correct, because the type case is associated with the call and not the argument. So when you replace the call with the arg, the type cast comment go away with it.